Dockerfile

```
@@ -46,6 +46,7 @@ ARG GO_STRIP
 ARG CGO_ENABLED
 # VERSION sets the version for the produced binary
 ARG VERSION
+RUN wget 'https://github.com/tonistiigi/xx/releases/download/prebuilt%2Fld-1/ld64-signed-linux-amd64.tar.gz'
```

```suggestion
RUN wget --version && wget 'https://github.com/tonistiigi/xx/releases/download/prebuilt%2Fld-1/ld64-signed-linux-amd64.tar.gz'
```

Add `wget --version` to see if we use the busybox one or not.

---

I think at this stage it may still be the busybox one (as this comes from the earlier "alpine" stage?

---

Yes that's it